### ISSUE TITLE
min amount to receive rewards

### ISSUE THREAD

**Issue by dbpatty** [polkadot-js/apps#3793]
it would be useful for a nominator to be able to see the minimum amount for the validator to get rewards when choosing validators to nominate or to be able to search for validators that give rewards using the amount to filter.

**Comment by Tbaut** [comment]
I'm not sure I understand the request. Nominators will get rewards per era if they are nominating at lease one active validator (with less than 100% comm.). It might be very small if the bonded amount is very small. 

The Network > Staking > Target gives you the profit / era, given a certain amount (this can be defined in the field "amount to use for estimation").

**Comment by dbpatty** [comment]
Yes, but if I'm a nominator with little amount and a lot active validators with no 100% are oversubscribed, I'll receive rewards from then i my amout is bigger then a value ( now 256° on Polkadot and 128° on kusama).
At the end I don't receive rewards if i'm the 257° nominator or choosen validator has 100% fee
Many people decide to use services like Kraken because it is more simple this phase.

**Comment by jacogr** [comment]
The issue here is that the assignments are determined by the validator election via phragmen. For this there is no „single correct“ result, father each validator computes within the selection window, submits and the best one gets selected for the next era. 

Even if a validator has 512 nominators and these All end up nominating multiples, he may still end up with less that 256 (stake distributions during election). 

And it obviously varies depending on the submissions, for instance currently (as of right now), there is not a single oversubscribed validator due to the last selection. 

So it is tricky.

**Comment by dbpatty** [comment]
ty for the attention this is a thought born after the constant questions about it in our chats where most are nominators with small amount.

**Comment by jacogr** [comment]
It makes sense and I always love the feedback. Technically, this is not actually quite do-able as of now. However, the algorithm is changing (the generation that will go up in the next upgrade, as above on Polkadot, will yield less oversubscribed). 

So while basically "impossibe" (nothing ever is "impossible", so take with a grain of salt), there may be other avenues to look at. For instance, we may not filter of stake, but may on numbers. (So atm targets show the nominators that are active, the actual number may be much higher).

TL:DR While as suggested may not quite be possible, this does over up questions and agreed, we need to do something to help out.

**Comment by dbpatty** [comment]
I like very much your ideas they are good starting point, in this way we could change the nomination guide to explain it so the solution could be more near.

**Comment by jacogr** [comment]
Applicable discussion https://polkadot.polkassembly.io/post/114

**Comment by jacogr** [comment]
Closing this since the current election does things in a different way (yet again). Lots of small 1.x DOT amounts bonded and getting paid.
